### PR TITLE
Initialize framework with application launch arguments

### PR DESCRIPTION
**Pull request by checkthebounds**

### CODE REVIEW

File: NimbleDroid/NimbleDroid.swift [hunk 1]
```diff
@@ -7,12 +7,46 @@
...
+            coldStartup = true
+            NSLog("NDScenario initialized pid %d coldStartup", processIdentifier)
+        } else {
+            NSLog("NDScenario initialized pid %d", processIdentifier)
+        }
+        initialized = true
+    }
+
     public class func begin(bookendID : String) {
+        if bookendID == scenario {
```
Comment (johneyoung) [inline NimbleDroid/NimbleDroid.swift]: Can we add some debug log if it's used w/o setup()?

**Review by johneyoung — COMMENTED** [review]
Left a comment. All look good other than it.

File: NimbleDroid/NimbleDroid.swift (hunk 1 above)
Comment (checkthebounds) [reply NimbleDroid/NimbleDroid.swift]: setup() produces a debug message when it *is* used so there's no need to add extra debug messages later.

Comment (johneyoung) [reply NimbleDroid/NimbleDroid.swift]: K, go ahead w/ what you think good. (But in general, logs are not bad even if they are redundant.)

Comment (checkthebounds) [reply NimbleDroid/NimbleDroid.swift]: Now that I think about it, if the user is testing the framework locally this debug message will let them know they've forgotten to initialize. It's not redundant after all :)

**Pull request merged**